crates/bazed-core/src/lib.rs

```diff
@@ -0,0 +1,51 @@
+#![forbid(unreachable_pub)]
+use color_eyre::Result;
```

Comment:
Do we use the default ordering we get from the rust tooling here?

If so fix pls.

Reply:
Let's not bother with that level of formatting in ways that does't really affect dev efficiency, code readability or anything like it, _and_ doesn't get managed by rustfmt 👀

Reply:
Order does get affected by rustfmt, and then I would argue never to split, because alphabetical is nicer than random.

Reply:
I agree with buffet.

Reply:
I would have vehemently argued against effort here, but rustfmt.toml saves the day. all poggers, done.